src/material-experimental/mdc-snack-bar/snack-bar.spec.ts

```diff
@@ -941,6 +948,14 @@ class BurritosNotification {
       @Inject(MAT_SNACK_BAR_DATA) public data: any) { }
 }
 
+/** Component with an aria-live element for testing ComponentPortal. */
+@Component({template: '<span aria-live="polite">Hello there.</span>'})
+class KenobiNotification {
```

Comment:
Can you give this class a descriptive name like `ComponentWithAriaLive`?

(similar below)